draft-ietf-taps-impl.md

```diff
@@ -585,6 +585,20 @@ remote endpoint that can be used to parse Messages.
 MessageFramer.MakeConnectionReady(Connection)
 ~~~
 
+Alternatively, if the framer implementation deems the candidate selected during racing unsuitable, it can signal this to the Connection. 
+
+~~~
+MessageFramer.FailCandidate(Connection)
```

Comment:
Two things here:
1. This should take an error, like `FailConnection`
2. I think this doesn't need to be distinct from `FailConnection` overall. If the place where the framer is running is during candidate racing, then its failing its view of the connection is only failing one candidate. I don't know if we should have the term "candidate" bubbling up into the API.

Reply:
Good call, really no need to have it as a distinct call!